src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -63,6 +64,9 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
         document.document_filename,
       );
     }
+    if (this.caseField.field_type) {
```

Comment:
is this check required? we always have a field type for a field isn't it?

Reply:
refactored code